packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -903,8 +903,6 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
     // If an update is made to pointer scrolling here, consider if the same
     // (or similar) change should be made in
     // ScrollPositionWithSingleContext.pointerScroll.
-    assert(delta != 0.0);
```

Comment:
In the case of 0, can we go idle and just return? Just in case allowing delta to be 0 in these `pointerScroll` implementations would introduce other regressions.